.github/workflows/reconcile-issue.yaml

```diff
@@ -0,0 +1,146 @@
+name: Reconcile Issue
+
+on:
+  workflow_call:
+
+jobs:
+  labels:
+    name: Check labels
+    runs-on: ubuntu-latest
+    if: github.event.action != 'closed'
+    steps:
+      - name: triage
+        uses: actions/github-script@v7
+        with:
+          script: |
+            // begin helper function for adding comments
```

Comment:
can this be a separate file. As inline YAML, it is harder to review the logic within it, because we lose syntax highlighting (which can make hard to maintain this in the future)

Reply:
I agree. I think what I will do is create an issue to rewrite it in golang to expand those that can help us maintain/fix it when it breaks.

Reply:
I'll write the issue for that after this merges so I can link to the source lines where we have javascript as inline YAML.